start.sh

```diff
@@ -272,15 +272,15 @@ while [[ $# -gt 0 ]]; do
         echo -e "  -d,   --debug\t\t\t Run in debug mode"
```

Comment:
I can give a comment on the line 250 and 270, but now --no-rebuild option is useless

Reply:
--no-rebuild deleted